prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -2,27 +2,55 @@
 
 layout: page_v2
 title: Custom or No mediation
-description: Integration of Prebid SDK withou primaty Ad Server
+description: How to integrate the Prebid SDK without a primary Ad Server
 sidebarType: 2
 
 ---
 
+
+
 # Custom Bidding Integration
 {:.no_toc}
 
-You can use Prebid SDK to monetize your app with a custom ad server or even without it. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without primary ad server and its SDK.
+## Before you start
+
+Before implementing the instructions in this guide, you need to ensure that you have correctly initialized the Prebid Mobile SDK in your app. The initialization method has the following pattern:
+
+```java
+Prebid.init(context, adUnit, accountID, adServer, host);
```

Comment:
@jsligh - why would adUnit be part of the initialization? This doesn't make sense to me. All other other params make sense, but there could be many adunits. Is the idea that they have to init for each and every adunit?

Reply:
looks like adunit was removed from the doc.